src/main/java/org/opentripplanner/apis/gtfs/model/PlanPageInfo.java

```diff
@@ -0,0 +1,91 @@
+package org.opentripplanner.apis.gtfs.model;
+
+import graphql.relay.ConnectionCursor;
+import java.time.Duration;
+import java.util.Objects;
+
+public class PlanPageInfo {
+
+  private final ConnectionCursor startCursor;
+  private final ConnectionCursor endCursor;
+  private final boolean hasPreviousPage;
+  private final boolean hasNextPage;
+  private final Duration searchWindowUsed;
+
+  public PlanPageInfo(
+    ConnectionCursor startCursor,
+    ConnectionCursor endCursor,
+    boolean hasPreviousPage,
+    boolean hasNextPage,
+    Duration searchWindowUsed
+  ) {
+    this.startCursor = startCursor;
+    this.endCursor = endCursor;
+    this.hasPreviousPage = hasPreviousPage;
+    this.hasNextPage = hasNextPage;
+    this.searchWindowUsed = searchWindowUsed;
+  }
+
+  public ConnectionCursor startCursor() {
+    return startCursor;
+  }
+
+  public ConnectionCursor endCursor() {
+    return endCursor;
+  }
+
+  public boolean hasPreviousPage() {
+    return hasPreviousPage;
+  }
+
+  public boolean hasNextPage() {
+    return hasNextPage;
+  }
+
+  public Duration searchWindowUsed() {
+    return searchWindowUsed;
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(
+      startCursor != null ? startCursor.getValue() : null,
+      endCursor != null ? endCursor.getValue() : null,
```

Comment:
```suggestion
      startCursor == null ? null : startCursor.getValue(),
      endCursor == null ? null : endCursor.getValue(),
```
This is nitpicking, but the above change is easier to read and avoid unnecessary negation.